modules/RTC/RTCUtils.js

```diff
@@ -869,17 +869,41 @@ class RTCUtils extends Listenable {
                 this.p2pPcConstraints
                     = JSON.parse(JSON.stringify(this.pcConstraints));
 
-                // Allows sending of video to be suspended if the bandwidth
-                // estimation is too low.
-                if (!options.disableSuspendVideo) {
-                    this.pcConstraints.optional.push(
-                        { googSuspendBelowMinBitrate: true });
+                const {
+                    forceSuspendVideo,
+                    disableSuspendVideo
+                } = options;
+                const suspendVideoEnabled
+                    = typeof forceSuspendVideo === 'undefined'
+                        || (typeof forceSuspendVideo === 'number'
+                            && Math.random() < forceSuspendVideo);
+                const disableSuspendVideoForJVBPC
+                    = Boolean(disableSuspendVideo);
```

Comment:
do we need both variables `forceSuspendVideo` and `disableSuspendVideo`? can't we just say that, to disable, `forceSuspendVideo` should be `0` ?  with both of them we have to handle all the valid/invalid interactions between the 2 (if i'm understanding this correctly)